float32 line_speed
float32 angle_target
float32 angle_current
uint8 xtl_flag
uint8 in_bridge